lwt/server/Http.re

```diff
@@ -14,7 +15,30 @@ type error_handler =
 
 let make_request_handler: Server.t('s, 'r, 'a, 'b) => request_handler =
   (server, ~closer, _client, reqd) => {
-    let req = Httpaf.Reqd.request(reqd);
+    let req = reqd |> Httpaf.Reqd.request;
+    let read_body = () => {
```

Comment:
It should be possible to define this method in a new module, perhaps `lwt/server/Request.re`, that we can reuse across the `Http.re` and `Https.re` modules.